src/Arcus.EventGrid.Testing/Logging/NoOpLogger.cs

```diff
@@ -1,11 +1,13 @@
 ﻿using System;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
 
 namespace Arcus.EventGrid.Testing.Logging
 {
     /// <summary>
     /// Represents an <see cref="ILogger"/> model without any implementation.
     /// </summary>
+    [Obsolete("Unused logger implementation with the same functionality as " + nameof(NullLogger) + " will be removed in future major")]
     public class NoOpLogger : ILogger
```

Comment:
It looks like the code-coverage reports leads us to types that we can mark as obsolete :)

Reply:
Yes, these two loggers in this project does not have much added-value and it seems too big a work to maintain them.